Geometry/HGCalCommonData/src/HGCalTypes.cc

```diff
@@ -38,3 +38,12 @@ int32_t HGCalTypes::packCellType6(int type, int cell) { return (type * faccell6_
 int32_t HGCalTypes::getUnpackedCellType6(int id) { return (id / faccell6_); }
 
 int32_t HGCalTypes::getUnpackedCell6(int id) { return (id % faccell6_); }
+
+int32_t HGCalTypes::layerType(int type) {
+  int types[5] = {HGCalTypes::WaferCenter,
```

Comment:
couldn't this be made `constexpr`, so that we can avoid building an array every time `layerType` is called?

Reply:
I shall do it in the next round. We need this PR to be merged urgently so that we can move forward